lib/tasks/consolidation.rake

```diff
@@ -0,0 +1,61 @@
+# We don't care much about the coverage of scaffolding files that are just used on the desktop during
+# consolidation and are going to be removed.
+# :nocov:
+namespace :consolidation do
+  desc "Copy a locale key for all locales from government-frontend to frontend"
```

Comment:
Also, if you're using zsh like I am, you'll need to escape the brackets for the task to work, e.g

```
rake consolidation:copy_translation\[travel_advice\]
```

Otherwise you get an error:

```
zsh: no matches found: consolidation:copy_translation[travel_advice]
```

Perhaps this too could be added to the task description. Perhaps as a usage statement?

Apparently `rake -D` show the whole [description](https://ruby.github.io/rake/Rake/DSL.html#method-i-desc) if it's over multiple lines.

Reply:
Hmm... I'm using zsh and I didn't need to escape the brackets. I know you need to put the whole thing in speechmarks if you're running it directly in kubernetes, but that's more about the colon I think. Let me try again locally and see what's what.

Reply:
I've updated to add a bigger description, and have put the rake command in speechmarks, which is a standard way to get them to work in kubernetes and also works for zsh.